### PR TITLE
This adds a file about Chandigarh

### PR DESCRIPTION
**Pull request by hsinghwm**
This is a WIP
- [ ] add things to do
- [ ] add places to eat

### CODE REVIEW

**Comment by rsatishwm** [comment]
Please elaborate on things to do and places to eat. I wanna try some good Punjabi cuisines

File: chandigarh-India.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Chandigarh India
+- City Beautiful.. Cleanest city in India
+- First planned city in India
+- Awesome Punjabi food and boys
+
```
Comment (rsatishwm) [inline chandigarh-India.md:5]: Please elaborate on the food. Like best restaurants and their best dishes to taste

File: chandigarh-India.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Chandigarh India
+- City Beautiful.. Cleanest city in India
```
Comment (rsatishwm) [inline chandigarh-India.md:2]: That is awesome!!

File: chandigarh-India.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+- First planned city in India
```
Comment (rsatishwm) [inline chandigarh-India.md:3]: Great to know!!

**Comment by adityavipul** [comment]
Paaji please delete this branch..

**Pull request merged**